<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>password</class>
 <widget class="QMainWindow" name="password">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>password</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>711</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>711</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{
background-color: #26A69A;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>279</x>
       <y>40</y>
       <width>241</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Dubai</family>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #004D40;</string>
     </property>
     <property name="text">
      <string>password generator</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>154</x>
       <y>120</y>
       <width>500</width>
       <height>561</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
border-radius: 10px;
background-color: rgb(236, 236, 236);
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radio_site">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>46</y>
        <width>180</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#424242;
color: black;</string>
      </property>
      <property name="text">
       <string>generate pass for site</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_pass">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>46</y>
        <width>173</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#424242;
color: black;</string>
      </property>
      <property name="text">
       <string>generate a password</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_site">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>140</y>
        <width>360</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 200);
color: black;</string>
      </property>
      <property name="placeholderText">
       <string> name of site:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_generate">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>400</y>
        <width>360</width>
        <height>35</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{
color: #EEEEEE;
background-color: #009688;
}</string>
      </property>
      <property name="text">
       <string>generate</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_username">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>190</y>
        <width>360</width>
        <height>35</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(200, 200, 200);
color: black;</string>
      </property>
      <property name="placeholderText">
       <string> username:</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>350</y>
        <width>360</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #D500F9;</string>
      </property>
      <property name="minimum">
       <number>8</number>
      </property>
      <property name="maximum">
       <number>16</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pass_length">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>290</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#424242;
color: black;</string>
      </property>
      <property name="text">
       <string>password length:</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>293</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(187, 222, 251);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="pass">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>480</y>
        <width>360</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#424242;
color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buddy">
       <cstring>fdfdsfdsfds</cstring>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_search">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>240</y>
        <width>360</width>
        <height>35</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #9C27B0;
color: #EEEEEE;
border-raduis: 10px;
</string>
      </property>
      <property name="text">
       <string>search user and password</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_search">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>90</y>
        <width>361</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>search password and username of a site</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_user">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>460</y>
        <width>361</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #EEEEEE;
background-color: #880E4F;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_password">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>500</y>
        <width>361</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #EEEEEE;
background-color: #880E4F;</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lb_password">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>505</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>password:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lb_username">
      <property name="geometry">
       <rect>
        <x>3</x>
        <y>466</y>
        <width>70</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Dubai</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="text">
       <string>username:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>lcdNumber</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>547</x>
     <y>441</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_site</sender>
   <signal>clicked()</signal>
   <receiver>le_site</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>183</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_site</sender>
   <signal>clicked()</signal>
   <receiver>le_username</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>le_site</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>btn_search</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>405</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>422</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>le_user</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>le_password</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>632</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>lb_username</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>590</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radio_search</sender>
   <signal>clicked()</signal>
   <receiver>lb_password</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
